dev-vcs/git/git-2.36.1.recipe

```diff
@@ -0,0 +1,301 @@
+SUMMARY="Fast, scalable, distributed revision control system"
+DESCRIPTION="Git is a free and open source distributed version control system \
+designed to handle everything from small to very large projects with speed \
+and efficiency.
+
+Git is easy to learn and has a tiny footprint with lightning fast performance. \
+It outclasses SCM tools like Subversion, CVS, Perforce, and ClearCase with \
+features like cheap local branching, convenient staging areas, and multiple \
+workflows."
+HOMEPAGE="https://git-scm.com/"
+COPYRIGHT="2005-2020 Git Authors (see git web site for list)"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://www.kernel.org/pub/software/scm/git/git-$portVersion.tar.xz"
+CHECKSUM_SHA256="405d4a0ff6e818d1f12b3e92e1ac060f612adcb454f6299f70583058cb508370"
+SOURCE_URI_2="https://www.kernel.org/pub/software/scm/git/git-manpages-$portVersion.tar.xz"
+CHECKSUM_SHA256_2="b1e6e651333283bfe2abbdf10baa858f61c5ec9d3caacb30f44888d78d964e6f"
+SOURCE_URI_3="https://www.kernel.org/pub/software/scm/git/git-htmldocs-$portVersion.tar.xz"
+CHECKSUM_SHA256_3="f2a44ee1acfc1b8e2ce9a90ae767226c862d5a6596bed0404de70fdef8e67c8d"
+PATCHES="git-$portVersion.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+GLOBAL_WRITABLE_FILES="
+	settings/gitconfig keep-old
+	"
+USER_SETTINGS_FILES="
+	settings/git directory
+	"
+
+PROVIDES="
+	git$secondaryArchSuffix = $portVersion compat >= 2.3
+	cmd:git = $portVersion compat >= 2.3
+	cmd:git_receive_pack = $portVersion compat >= 2.3
+	cmd:git_shell = $portVersion compat >= 2.3
+	cmd:git_upload_archive = $portVersion compat >= 2.3
+	cmd:git_upload_pack = $portVersion compat >= 2.3
+	"
+REPLACES="
+	git
+	"
```

Comment:
This seems to do the trick here @korli , checked with pkgman to install git and got a conflict with the installed git_x86

Reply:
it's only for x86_gcc2. please add a if ...

Reply:
Right, wouldn't work on x86_64 ;)